rpm/rust-cbindgen.spec

```diff
@@ -0,0 +1,70 @@
+%global rustflags -Clink-arg=-Wl,-z,relro,-z,now
+
+Name:           cbindgen
+Version:        0.16.0
+Release:        0
+Summary:        A tool for generating C bindings from Rust code
+License:        MPL-2.0
+URL:            https://crates.io/crates/cbindgen
+Source:         %{name}-%{version}.tar.bz2
+BuildRequires:  cargo >= 1.30.0
+BuildRequires:  rust >= 1.30.0
+BuildRequires:  rust-std-static >= 1.30.0
+
+%description
+A tool for generating C bindings from Rust code.
+
```

Comment:
Then we could introduce the package for the cbindgen as...

```
%package     -n %{crate}
Summary:        %{summary}
License:        MPLv2.0 and MIT and BSD and (ASL 2.0 or Boost)
```

Reply:
See https://src.fedoraproject.org/rpms/rust-cbindgen/blob/master/f/rust-cbindgen.spec